webapp/channels/src/components/access_history_modal/access_history_modal.test.tsx

```diff
@@ -38,26 +40,23 @@ describe('components/AccessHistoryModal', () => {
         expect(wrapper.find(AuditTable).exists()).toBe(true);
     });
 
-    test('should have called actions.getUserAudits when onShow is called', () => {
+    test('should have called actions.getUserAudits only when first rendered', () => {
         const actions = {
             getUserAudits: jest.fn(),
         };
         const props = {...baseProps, actions};
-        const wrapper = shallow<AccessHistoryModal>(
-            <AccessHistoryModal {...props}/>,
-        );
+        const view = render(withIntl(<AccessHistoryModal {...props}/>));
```

Comment:
`shallow` does not run the effects, so we move on with `render` from the `testing-library/react`.